examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -102,6 +102,27 @@ typedef Particles<ParticleDescription<
 
 /*--------------------------- ions -------------------------------------------*/
 
+/*! Specify (chemical) element
+ * 
+ * Proton and neutron numbers define the chemical element that the ion species
+ * is based on. This value can be non-integer for physical models taking  
+ * charge shielding effects into account. @see http://en.wikipedia.org/wiki/Effective_nuclear_charge
+ * 
+ * It is wrapped into a struct because of C++ restricting floats from being 
+ * template arguments. 
+ * 
+ * Do not forget to set the correct mass of the atom in @see physicalConstants.param !
+ */
+struct NumberOfProtons
+{
+    static const float_X value = 2.0;
+};
+
+struct NumberOfNeutrons
```

Comment:
yes, basically the `deselect` must be able to work for const -> non-const attributes.

the "element" name was just given as an example, I think our users should be able to read periodic tables to set helium1 and helium2 in there simulation :) of course, we can write a nice list of examples in our wiki or elsewhere, too :)